tests/integration/cli/test_accounts.py

```diff
@@ -163,20 +163,96 @@ def test_generate(ape_cli, runner, temp_keyfile_path):
     result = runner.invoke(
         ape_cli,
         ["accounts", "generate", ALIAS],
-        input="\n".join(["random entropy", PASSWORD, PASSWORD]),
+        input="\n".join([PASSWORD, PASSWORD, "random entropy"]),
```

Comment:
Change this back

Reply:
Wait, does this ask you for entropy internally? I didn't think it did

Reply:
no it doesnt so I removed it.

Reply:
On second thought, then we would need to move passphrase to before entropy to keep the previous passphrase passphrase entropy workflow

Reply:
resolved this in cli and tests